docs/source/alfacase/api_reference.rst

```diff
@@ -12,6 +12,8 @@ Case
 
 .. autoclass:: alfasim_sdk.CaseDescription()
 
+.. autoclass:: alfasim_sdk.AlfasimVersionInfo()
```

Comment:
https://alfasim-sdk--373.org.readthedocs.build/en/373/alfacase/api_reference.html#alfasim_sdk.CaseDescription

![image](https://github.com/ESSS/alfasim-sdk/assets/4542185/f363085a-e737-480d-b1dd-00885e24f366)

The generated docs is missing a link for `AlfasimVersionInfo` in `CaseDescription` definition =(
At first look I didn't notice anything missing. Could you recheck that?